Tests/UnitTests/Views/ListViewTests.cs

```diff
@@ -1157,4 +1158,98 @@ void Lw_CollectionChanged (object sender, NotifyCollectionChangedEventArgs e)
             }
         }
     }
+    [Fact]
```

Comment:
I believe these tests have no dependency on `Application` or `ConfigurationManager` and thus can be moved to parallizable? If I'm right, please do so.